packages/css-data/bin/mdn-data.ts

```diff
@@ -49,6 +49,23 @@ const normalizedValues = {
   "text-size-adjust": autoValue,
 } as const;
 
+const beautifyKeyword = (property: string, keyword: string) => {
+  if (keyword === "currentcolor") {
+    return "currentColor";
+  }
+  // builder style panel cannot interpret "normal" and "bold"
+  // always expected numeric value
+  if (property === "font-weight") {
```

Comment:
this is suspicious for bugs
- there is more keywords
-shouldn't the color input be able to interpret those keywords correctly?

Reply:
font-weight control. Yes it should, but we can fix it later. Its logic is not trivial.

Reply:
got it, lets create an issue, add everything you know that needs to happen, add the link here in a todo